src/main/java/training/busboard/ApiServices/TflApiRadiusResponseService.java

```diff
@@ -1,14 +1,17 @@
 package training.busboard.ApiServices;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import training.busboard.Models.StopPoint;
 
 import java.util.List;
 
 @JsonIgnoreProperties(ignoreUnknown=true)
 public class TflApiRadiusResponseService {

```

Comment:
This comes back to what we were discussing yesterday about the difference between different types of objects - this is called `Service` but it doesn't do any of the work of looking up the stop points. It's really just a value object containing a list of `StopPoint`s, so I'd name it like that. Maybe `StopPoints` or `StopPointResult`.